code/datums/noobmode.dm

```diff
@@ -0,0 +1,129 @@
+/datum/action/equipHazard
+	name = "Equip skinsuit"
+	desc = "hazard gear and internals. Takes a few seconds."
+	icon_icon = 'icons/obj/clothing/suits.dmi'
+	button_icon_state = "skinsuit"
+	var/obj/item/clothing/suit/space/skinsuit/suitslot
+	var/obj/item/clothing/mask/breath/maskslot
+	var/obj/item/clothing/head/helmet/space/helmetslot
+	var/obj/item/tank/internals/airslot
+	var/obj/item/storage/box/survBox
+	var/paniced = FALSE
+
+/datum/action/equipHazard/Trigger()
+	if(ishuman(owner))
+		var/mob/living/carbon/human/H = owner
+		testing(paniced)
+		if(!paniced)
+			if(pannickEquip(H))
```

Comment:
"pannick"?

Reply:
Ill fix the spelling when iam home again